app/src/main/java/com/pac_man/GameInitializer.java

```diff
@@ -0,0 +1,145 @@
+import com.pac_man.characters.Ghost.ClydeChaser;
+import com.pac_man.characters.Ghost.InkyChaser;
+import com.pac_man.characters.Ghost.PinkyChaser;
+import com.pac_man.characters.Pacman.Pacman;
+
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameInitializer {
+    public void init() {
+        URL pacmanImg = getClass().getResource("app/src/main/java/com/pac_man/Resources/PacMan/pr1.png");
+
+        Path pacmanImgPath = null;
+        try {
+            pacmanImgPath = Paths.get(pacmanImg.toURI());
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
+        }
+
+        Sound pacmanSound = new Sound(null); // sounds aren't implemented
+
+        Sprite pacManSprite = new Sprite(
+                new Coord(0, 0),
+                new Size(10.0, 10.0),
+                pacmanImgPath
+        );
+
+        Pacman pacman = new Pacman(
+                new Position(0, 0),
+                pacManSprite,
+                new KeyboardListener()
+        );
+
+        List<Ghost> ghosts = getGhosts(pacman);
+
+        for (Ghost ghost : ghosts) {
+            ghost.spawn();
+        }
+
+        Maze maze = new Maze();
+    }
+
+
+    private List<Ghost> getGhosts(Pacman pacman) {
+
+        URL blinkyImg = getClass().getResource("app/src/main/java/com/pac_man/Resources/Ghosts/Blinky/red1.png");
+        URL clydeImg = getClass().getResource("app/src/main/java/com/pac_man/Resources/Ghosts/Clyde/yell1.png");
+        URL inkyImg = getClass().getResource("app/src/main/java/com/pac_man/Resources/Ghosts/Inky/blue1.png");
+        URL pinkyImg = getClass().getResource("app/src/main/java/com/pac_man/Resources/Ghosts/Pinky/pink1.png");
+
+
+        Path blinkyImgPath = null;
+        Path clydeImgPath = null;
+        Path inkyImgPath = null;
+        Path pinkyImgPath = null;
+
+        try {
+            blinkyImgPath = Paths.get(blinkyImg.toURI());
+            clydeImgPath = Paths.get(clydeImg.toURI());
+            inkyImgPath = Paths.get(inkyImg.toURI());
+            pinkyImgPath = Paths.get(pinkyImg.toURI());
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
+        }
+
+        Sprite blinkySprite = new Sprite(
+                new Coord(0, 0),
+                new Size(10.0, 10.0),
+                blinkyImgPath
+        );
+
+        Sprite clydeSprite = new Sprite(
+                new Coord(0, 0),
+                new Size(10.0, 10.0),
+                clydeImgPath
+        );
+
+        Sprite inkySprite = new Sprite(
+                new Coord(0, 0),
+                new Size(10.0, 10.0),
+                inkyImgPath
+        );
+
+        Sprite pinkySprite = new Sprite(
+                new Coord(0, 0),
+                new Size(10.0, 10.0),
+                pinkyImgPath
+        );
+
+
+
+        Maze maze = new Maze();
```

Comment:
Maze its not used., for now are creating the classes, but I think  its needed to define a loop to manage the logic of the game, because for now the class only instance Sprites, in other words only draw on the console